packages/devtools/lib/src/server_api_client.dart

```diff
@@ -0,0 +1,87 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:convert';
+import 'dart:html';
+
+import 'package:sse/client/sse_client.dart';
+
+class ServerApiClient {
+  ServerApiClient() : _channel = SseClient('/api/sse') {
```

Comment:
to enable using this in a debug workflow we should not hardcode it to assume the SseClient is on the same hostname as the app.
If you add a query parameter to devtools passing in the hostname of the server then it will be easier to test this with debug builds if people need to. That will matter more when we make the server vend the analysis server protocol. We also need to pass in a security token like what the Dart VMService does to the SseClient to avoid a security vulnerability.